### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by adictn** [FrostiiZ/fb-messenger-official-dark-mode#1]
The page loading becomes very slow, the left sidebar is dark, but the main chat in the middle is still white.

**Comment by FrostiiZ** [comment]
Hi ! Thank you for reporting this issue, I've not been using messenger.com for quite some time, thus I haven't noticed if the script was still working or not. I managed to make it work again and I pushed an update.

It seems like Meta made some changes on how the page is stylized.

I made some tweaking and my userscript is working again. I have pushed an update, look into your script manager if it pulled the latest 0.3 or update it manually.

Meta now forces the light theme class onto almost every page node instead of only the main body element, which means every element that is updated is loaded with its default light class. Any change in the conversation view would subsequently break the dark theme, like switching to a different chat.

My userscript now looks for all these page changes and sets back the dark theme everywhere every time, so switching conversations would still apply the dark theme.
This might result in some overhead but on my end it looks like even without my script the page is still loading slowly.

I'm sorry about the flashbang when you open messenger.com, as of now I'm not able to set the theme to dark before content is being loaded completely.
I still hope this is useful to you and thank you for using my script !
